src/plugins/intel_cpu/src/nodes/random_uniform.hpp

```diff
@@ -12,17 +12,30 @@ namespace ov {
 namespace intel_cpu {
 namespace node {
 
+// Following const values are taken from the original paper:
+// https://www.thesalmons.org/john/random123/papers/random123sc11.pdf
+constexpr uint32_t CRUSH_RESISTANCE_CONST_LOWER_VALUE = 0x9E3779B9;
```

Comment:
I believe we don't need to define these constants for other objects. Please return them to cpp.

Reply:
Done!